docs/design/chart_format.md

```diff
@@ -189,25 +176,14 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: GoTemplates
```

Comment:
ok, I think it is important to keep existing charts running.
Otherwise, why merge them ?

if we can pass through when no expander is specified then fine. but this needs to work in MVP.

so LGTM